### ISSUE TITLE
geom_violinhalf and geom_violindot not freeing scales with facet_wrap()

### ISSUE THREAD

**Issue by alspeed09** [easystats/see#81]
Hi!

I installed today "see" from CRAN (v0.6.0) [12/05/2020]

I've been trying to plot a half-violin plot using facet_wrap() while freeing the y scale. It works with the regular violin plot, but not with the geom_violinhalf or geom_violindot. Here is a reproducible example:

```
library(tidyverse)
library(see)

iris %>%
    pivot_longer(cols = 1:4, names_to = "variable", values_to = "values") %>%
    ggplot(aes(x = Species, y = values, fill = Species)) +
    geom_violin() +
    facet_wrap(~ variable, scales = "free_y") +
    theme_bw()
```

Which produces this output with the scale free on the y-axis
![IssueImage1](https://user-images.githubusercontent.com/13182678/101235414-72908400-3696-11eb-824a-244fc32540e4.png)

While the code using geom_violinhalf or geom_violindot doesn't do that:

```
iris %>%
    pivot_longer(cols = 1:4, names_to = "variable", values_to = "values") %>%
    ggplot(aes(x = Species, y = values, fill = Species)) +
    geom_violinhalf() +
    facet_wrap(~ variable, scales = "free_y") +
    theme_bw()
```

![IssueImage2](https://user-images.githubusercontent.com/13182678/101235430-92c04300-3696-11eb-9793-76f6df33715e.png)

**Comment by strengejacke** [comment]
@DominiqueMakowski maybe you have an idea?

**Comment by DominiqueMakowski** [comment]
Not really, though I wonder if he should move to a less hacky way of doing half violins (for instance using [ggdist](https://github.com/mjskay/ggdist)), it would definitly be more robust
